packages/adapter/src/lib/adapter/adapter.ts

```diff
@@ -11580,28 +11576,51 @@ export class AdapterClass extends EventEmitter {
 
                         this.oStates = _states;
                         this.subscribeStates('*');
+
                         if (this._firstConnection) {
                             this._firstConnection = false;
-                            typeof this._options.ready === 'function' && this._options.ready();
-                            this.emit('ready');
+                            this._callReadyHandler();
                         } else {
-                            typeof this._options.reconnect === 'function' && this._options.reconnect();
+                            // TODO: is this really needed? If yes, we should document it.
```

Comment:
Remove it. it has no effect since redis DB ... And we have a major releasem so best idea. Kill it